kiwi/boot/image/builtin_kiwi.py

```diff
@@ -165,7 +166,7 @@ def create_initrd(self, mbrid=None, basename=None, install_initrd=False):
             # during the process of creating an image but not when the
             # image is actually booting with this initrd
             exclude_from_archive = [
-                '/' + Defaults.get_shared_cache_location(),
+                '/' + defaults.SHARED_CACHE_DIR,
```

Comment:
I would keep the get_shared_cache_location so these files remain exactly the same and the change is transparent.

Reply:
yes, all of these calls to get_shared_cache_location should stay

Reply:
@jesusbv sorry if I confused you with my first suggestion how to access the default variable. For compatibility and transparency the method get_shared_cache_location should stay, such that all code parts that calls it also stays untouched. The method itself changes and needs to do the following:

* strip an eventual leading / from the SHARED_CACHE_DIR variable
* return that value